jsonrpc/txpool_endpoint.go

```diff
@@ -41,6 +46,37 @@ type StatusResponse struct {
 	Queued  uint64 `json:"queued"`
 }
 
+// ContentFrom returns the transactions contained within the transaction pool.
+func (t *TxPool) ContentFrom(addr types.Address) (interface{}, error) {
+	convertTxMap := func(txMap []*types.Transaction) map[uint64]*transaction {
+		result := make(map[uint64]*transaction, len(txMap))
+		for key, tx := range txMap {
+			result[uint64(key)] = toTransaction(tx, nil, &types.ZeroHash, nil)
```

Comment:
Read and set the current header information (block number):
https://github.com/ethereum/go-ethereum/blob/b393ad8d29fe002fe6c0329a09d7715b00030c79/internal/ethapi/api.go#L203.

Also are we returning signed transaction (namely are V, R and S fields populated as well as `From` field)? 

If not, take a look to this logic (they need block number and active forks in order to create the appropriate signer and then recover sender, based on signature and hash):
https://github.com/ethereum/go-ethereum/blob/b393ad8d29fe002fe6c0329a09d7715b00030c79/internal/ethapi/api.go#L1333-L1335